gen-pydoc.py

```diff
@@ -0,0 +1,45 @@
+import pydoc
+import pkgutil
+import importlib
+import os
+import sys
+import click
+
+def generate_docs(package_name, output_file=None, file_prefix=None):
+    package = importlib.import_module(package_name)
+    
+    out_f = open(output_file, 'w') if output_file else sys.stdout
+    
+    out_f.write(f"\n# pydoc documentation for package: {package_name}\n")
+    out_f.write("\n---\n")
+    
+    for _, module_name, is_pkg in pkgutil.walk_packages(package.__path__, package.__name__ + "."):
+        out_f.write(f"\n## pydoc of module: {module_name}\n")
+        out_f.write("\n---\n")
```

Comment:
Did you check how it works in cases where there's a module-level import from another module?
e.g. most of the things listed here: https://github.com/DagsHub/client/blob/master/dagshub/__init__.py
are imported from another module, so it could be documenting those functions two times

Reply:
Everything is printed only once when I run it on the dagshub module